session/channel.go

```diff
@@ -226,18 +226,22 @@ func makeChUpdateNotif(currChInfo perun.ChInfo, proposedState *pchannel.State, e
 }
 
 // SubChUpdates implements chAPI.SubChUpdates.
-func (ch *Channel) SubChUpdates(notifier perun.ChUpdateNotifier) error {
+func (ch *Channel) SubChUpdates(notifier perun.ChUpdateNotifier) perun.APIErrorV2 {
 	ch.Debug("Received request: channel.SubChUpdates")
```

Comment:
Can you also update the log message to include the method name as a field. You can use your previous PR  (#153) as reference.